src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TeamsController.cs

```diff
@@ -251,13 +253,15 @@ public JsonResult Delete(int id)
             return Json(result, JsonRequestBehavior.DenyGet);
         }
 
+#pragma warning disable S2360 // Optional parameters should not be used
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)